from builtins import Exception


class InstantiateCSVError(Exception):
    """
    Создание экземпляра класса нового исключение InstantiateCSVError.

    """
